media_driver/agnostic/common/codec/shared/codec_def_encode_avc.h

```diff
@@ -860,6 +860,24 @@ typedef struct _CODEC_AVC_ENCODE_PIC_PARAMS
     */
     bool            bDisableFrameSkip;
 
+    /*! \brief Maximum frame size for all frame types in bytes.
+    *
+    *    Applicable for CQP and multi PAK. If dwMaxFrameSize > 0, driver will do multiple PAK and adjust QP (frame level QP + slice_qp_delta) to make the compressed frame size to be less than this value. If dwMaxFrameSize equals 0, driver will not do multiple PAK and do not adjust QP.
```

Comment:
could you help to reformat the comments, it's too long to be in a single line

Reply:
Yes, I will reformat the comments.